src/fiskaltrust.Launcher/Extensions/LifetimeExtensions.cs

```diff
@@ -28,6 +32,27 @@ public static IHostBuilder UseCustomHostLifetime(this IHostBuilder builder)
 #pragma warning restore CA1416
                 });
             }
+            else if (isSystemd)
```

Comment:
Do you know/have you tried using [`SystemdHelpers.IsSystemdService()`](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.hosting.systemd.systemdhelpers.issystemdservice?view=dotnet-plat-ext-8.0#microsoft-extensions-hosting-systemd-systemdhelpers-issystemdservice)? That would be a bit easier and cleaner I think 😁